Aggiunta mappa con regioni + fix grafico con dark mode.

lollo03 opened the pull request:
Vedi https://lollo03.github.io/quanto-manca/.
Fix dark mode:

https://user-images.githubusercontent.com/23106263/104782289-42073400-5784-11eb-986e-43bddaf49fe6.mp4

lollo03 committed:
Nuova mappa

Aggiunta mappa che mostra quanti giorni mancano in ciascuna regione

lollo03 committed:
Create test-build

lollo03 committed:
Rename test-build to test-build.yml

lollo03 committed:
Disattivate web Analystics per evitare casini

lollo03 committed:
Delete test-build.yml

lollo03 committed:
Update template.html

lollo03 committed:
Update main.py

MarcoBuster commented:
Non mi sembra un dato rilevante quello del "quanti giorni mancano" per ogni regione, visto che si adeguereanno tutte. 
E' più rilevante secondo me il dato della percentuale delle dosi somministrate rispetto alle dosi consegnate. Che dicI?

lollo03 commented:
È possibile integrare anche quel dato, anche se credo che bisogni guardare anche un altro dataset.

MarcoBuster commented:
@lollo03 Il dato c'è già nel dataset ufficiale

lollo03 commented:
Apposto, aggiungo questo dato ed aggiorno tutto

lollo03 committed:
Aggiunto CDN per dipendenze mappa.

lollo03 committed:
Cambiamenti alla mappa

Aggiunto il dato della % delle dosi somministrate, ed ora la mappa usa quel valore come riferimento per i colori.

lollo03 committed:
Fix dark mode

MarcoBuster commented:
Grazie. Faccio la review domani.

MarcoBuster commented:
@lollo03 potresti scrivermi su Telegram, per favore?
